vendor.conf

```diff
@@ -6,7 +6,7 @@ github.com/davecgh/go-spew v1.1.0
 github.com/pmezard/go-difflib v1.0.0
 golang.org/x/sys 314a259e304ff91bd6985da2a7149bbf91237993
 
-github.com/containerd/containerd e1428ef05460da40720d622c803262e6fc8d3477
+github.com/containerd/containerd 437e90aa5e6ab061ab9ea14845336d2f1b900fb0
```

Comment:
Can you pull through the relevant changes made to `github.com/containerd/containerd/vendor.conf` by this while you are at it please, specifically the `github.com/containerd/console` bump.